patterns/vsi-extension/README.md

```diff
@@ -5,3 +5,23 @@ This logic creates a VSI to an existing landing zone VPC.
 This code creates and configures the following infrastructure:
 - Adds an SSH key to IBM Cloud or uses an existing one.
 - Adds a VSI in each subnet of the landing zone VPC.
+
+There are two ways through which a user can pass the VPC details for deploying the VSI, both the approaches are mutually exclusive.
+
+## Using `vpc_id`
+
+The VPC ID of the landing zone VPC can be assigned to the variable vpc_id in order to create a VSI within that specific VPC.
+
+## Using `prerequisite_workspace_id` and `existing_vpc_name`
+
+The user can specify the workspace ID associated with the deployment of the landing zone VPC when creating a new VSI.
+
+Follow these steps to get the schematics workspace ID.
+
+1. Click the Navigation menu icon, and then click Schematics > Workspaces.
+1. Select the Workspace that is associated with landing zone VPC.
+1. Click the Settings.
+1. In the Details section, you can find the Workspace ID.
```

Comment:
I don't think consumer need to do this - my understanding is that when consuming from catalog, a dropdown will appear with a list of schematic workspaces, but I'm wondering if that only works if we have the `schematics_workspace_id` output. I'll start up a thread with catalog team to confirm. Either way, we will want to import this into catalog to test it. I can work with you on that

Reply:
Actually, sorry like you said, we already have this output in the VPC pattern, so I think what we should next is merge this code, and create an entry in our private catalog to test